### ISSUE TITLE
2019-11-20 Interpolation still occurs when mct$INCLUDEINTERPOLATION == 0

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#46]
### Issue

When setting `INCLUDEINTERPOLATION` in the `mct` to 0, interpolation still occurs. This has been tested and found to occur for tc010, tc011, tc012, tc013, tc014 (using historic commit 066fa5d8a1), tc015, tc016. Testing of test cases without interpolation requested in the mct were changed to request a partial AUC with interpolation.

```
mct$INCLUDEINTERPOLATION <- 1
results_list <- run_computation(data = d, map = mct, flag = flags, 
  parameterset = parameterset)
r_on <- results_list$data_out
e_on <- results_list$est_data
f_on <- results_list$flag_data

mct$INCLUDEINTERPOLATION <- 0
results_list <- run_computation(data = d, map = mct, flag = flags, 
  parameterset = parameterset)
r_off <- results_list$data_out
e_off <- results_list$est_data
f_off <- results_list$flag_data

all.equal(r_on, r_off)
all.equal(e_on, e_off)
all.equal(f_on, f_off)
```

```
> all.equal(r_on, r_off)
[1] TRUE
> all.equal(e_on, e_off)
[1] TRUE
> all.equal(f_on, f_off)
[1] TRUE
```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
There's still a problem with INCLUDEINTERPOLATION.

While it appropriately doesn't include the interpolated result when set off (INTERPOLATION==0) in the estimated concentation dataset, it does still generate AUCs that include interpolated endpoints.

```
j <- parameter_required("^AUC.[0-9]+?.", names(mct), simplify=FALSE)
mct[, names(j)]

(tclabel <- paste0("results.", testcase.id, ".", mct$MODEL, ".", mct$DOSINGTYPE, ".", mct$AUCMETHOD))

mct$INCLUDEINTERPOLATION <- 1
results_list <- run_computation(data = d, map = mct, flag = flags, 
                                parameterset = parameterset)
r_on <- results_list$data_out
e_on <- results_list$est_data
f_on <- results_list$flag_data

mct$INCLUDEINTERPOLATION <- 0
results_list <- run_computation(data = d, map = mct, flag = flags, 
                                parameterset = parameterset)
r_off <- results_list$data_out
e_off <- results_list$est_data
f_off <- results_list$flag_data

all.equal(r_on, r_off)
all.equal(e_on, e_off)
all.equal(f_on, f_off)

e_on
e_off

summary(r_off$AUC0_64)
summary(r_off$AUC60_124)

```

The results are as follows:

```
> j <- parameter_required("^AUC.[0-9]+?.", names(mct), simplify=FALSE)
> mct[, names(j)]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2 AUC.5.T1 AUC.5.T2 AUC.6.T1 AUC.6.T2
1        0       12        8       48        0       64       12      130       24       96       60      124
> 
> (tclabel <- paste0("results.", testcase.id, ".", mct$MODEL, ".", mct$DOSINGTYPE, ".", mct$AUCMETHOD))
[1] "results.tc010.M1.SD.LINLOG"
> 
> mct$INCLUDEINTERPOLATION <- 1
> results_list <- run_computation(data = d, map = mct, flag = flags, 
+                                 parameterset = parameterset)
unit_conversion : Unit Class 1 (Time) time_col:  TMAX TMIN TLAST TLAG KELTMLO KELTMHI THALF LASTTIME MRTLAST MRTEVIFO MRTEVIFP  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VZFO VZFP  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  CMAX CMIN CLAST CEST  parameters are scaled from  NG/ML  to  NG/ML  via scaling factor:  1 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLFO CLFP  parameters are scaled from  ML/HR  to  L/HR  via scaling factor:  0.001 
unit_conversion : Unit Class 8: (Amount.Time/Volume) auc_col:  AUCALL AUCLAST AUCLASTC AUCINFO AUCINFP AUCINFOC AUCINFPC  parameters are scaled from  NG.HR/ML  to  NG.HR/ML  via scaling factor:  1 
unit_conversion : Unit Class 9: (Amount.Time.Time/Volume) aumc_col:  AUMCLAST AUMCINFO AUMCINFP  parameters are scaled from  NG.HR.HR/ML  to  NG.HR.HR/ML  via scaling factor:  1 
unit_conversion : Unit Class 10: ([Amount.Time/Volume]/Amount) aucdn_col:  AUCALLDN AUCLASTDN AUCINFODN AUCINFPDN  parameters are scaled from  NG.HR/ML/MG  to  NG.HR/ML/MG  via scaling factor:  1 
unit_conversion : Unit Class 12: ([Amount/Volume]/Amount) concdn_col:  CMAXDN  parameters are scaled from  NG/ML/MG  to  NG/ML/MG  via scaling factor:  1 
unit_conversion : Unit Class 14: (Volume/Body Weight) volumew_col:  VZFOW VZFPW  parameters are scaled from  ML/KG  to  L/KG  via scaling factor:  0.001 
unit_conversion : Unit Class 15: (Volume/Time/Body Weight) clw_col:  CLFOW CLFPW  parameters are scaled from  ML/HR/KG  to  ML/HR/KG  via scaling factor:  1 
Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
3: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'AMOUNTU' #2# is not present in the dataset provided via 'map'
4: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU' and 'CONCU' #10# are not present in the dataset provided via 'map'
> r_on <- results_list$data_out
> e_on <- results_list$est_data
> f_on <- results_list$flag_data
> 
> mct$INCLUDEINTERPOLATION <- 0
> results_list <- run_computation(data = d, map = mct, flag = flags, 
+                                 parameterset = parameterset)
unit_conversion : Unit Class 1 (Time) time_col:  TMAX TMIN TLAST TLAG KELTMLO KELTMHI THALF LASTTIME MRTLAST MRTEVIFO MRTEVIFP  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VZFO VZFP  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  CMAX CMIN CLAST CEST  parameters are scaled from  NG/ML  to  NG/ML  via scaling factor:  1 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLFO CLFP  parameters are scaled from  ML/HR  to  L/HR  via scaling factor:  0.001 
unit_conversion : Unit Class 8: (Amount.Time/Volume) auc_col:  AUCALL AUCLAST AUCLASTC AUCINFO AUCINFP AUCINFOC AUCINFPC  parameters are scaled from  NG.HR/ML  to  NG.HR/ML  via scaling factor:  1 
unit_conversion : Unit Class 9: (Amount.Time.Time/Volume) aumc_col:  AUMCLAST AUMCINFO AUMCINFP  parameters are scaled from  NG.HR.HR/ML  to  NG.HR.HR/ML  via scaling factor:  1 
unit_conversion : Unit Class 10: ([Amount.Time/Volume]/Amount) aucdn_col:  AUCALLDN AUCLASTDN AUCINFODN AUCINFPDN  parameters are scaled from  NG.HR/ML/MG  to  NG.HR/ML/MG  via scaling factor:  1 
unit_conversion : Unit Class 12: ([Amount/Volume]/Amount) concdn_col:  CMAXDN  parameters are scaled from  NG/ML/MG  to  NG/ML/MG  via scaling factor:  1 
unit_conversion : Unit Class 14: (Volume/Body Weight) volumew_col:  VZFOW VZFPW  parameters are scaled from  ML/KG  to  L/KG  via scaling factor:  0.001 
unit_conversion : Unit Class 15: (Volume/Time/Body Weight) clw_col:  CLFOW CLFPW  parameters are scaled from  ML/HR/KG  to  ML/HR/KG  via scaling factor:  1 
Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
3: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'AMOUNTU' #2# is not present in the dataset provided via 'map'
4: In unit_conversion(data = data_data, map = map_data, result = computation_df,  :
  'PKPTMU' and 'CONCU' #10# are not present in the dataset provided via 'map'
> r_off <- results_list$data_out
> e_off <- results_list$est_data
> f_off <- results_list$flag_data
> 
> all.equal(r_on, r_off)
[1] "Component “AUC0_64”: Mean relative difference: 0.03165055"  "Component “AUC60_124”: Mean relative difference: 0.4093488"
> all.equal(e_on, e_off)
 [1] "Attributes: < Component “row.names”: Lengths (417, 321) differ (string compare on first 321) >"
 [2] "Attributes: < Component “row.names”: 318 string mismatches >"                                  
 [3] "Component “PKDATAROWID”: Numeric: lengths (417, 321) differ"                                   
 [4] "Component “SDEID”: Numeric: lengths (417, 321) differ"                                         
 [5] "Component “TIME”: Numeric: lengths (417, 321) differ"                                          
 [6] "Component “CEST_KEL”: Numeric: lengths (417, 321) differ"                                      
 [7] "Component “CEST_INT”: Numeric: lengths (417, 321) differ"                                      
 [8] "Component “CEST_EXT”: Numeric: lengths (417, 321) differ"                                      
 [9] "Component “CEST_C0”: Numeric: lengths (417, 321) differ"                                       
[10] "Component “CEST_TLAST”: Numeric: lengths (417, 321) differ"                                    
> all.equal(f_on, f_off)
[1] TRUE
> 
> e_on
    PKDATAROWID     SDEID TIME     CEST_KEL    CEST_INT     CEST_EXT CEST_C0 CEST_TLAST
1      22289754   2925168   24   1.70263761          NA           NA      NA         NA
2      22289755   2925168   48   0.58780724          NA           NA      NA         NA
7            NA   2925168   60           NA  0.39050000           NA      NA         NA
4            NA   2925168   64           NA  0.32866667           NA      NA         NA
3      22289756   2925168   72   0.20293065          NA           NA      NA 0.20293065
6            NA   2925168   96           NA          NA 7.005842e-02      NA         NA
8            NA   2925168  124           NA          NA 2.025778e-02      NA         NA
5            NA   2925168  130           NA          NA 1.552816e-02      NA         NA
11     22289571  40876220   16   0.49544432          NA           NA      NA         NA
21     22289572  40876220   24   0.36517471          NA           NA      NA         NA
31     22289573  40876220   48   0.14622386          NA           NA      NA         NA
81           NA  40876220   60           NA  0.08985000           NA      NA         NA
51           NA  40876220   64           NA  0.08246667           NA      NA         NA
41     22289574  40876220   72   0.05855120          NA           NA      NA 0.05855120
71           NA  40876220   96           NA          NA 2.344517e-02      NA         NA
9            NA  40876220  124           NA          NA 8.059791e-03      NA         NA
61           NA  40876220  130           NA          NA 6.411404e-03      NA         NA
12     22289531  63536292   12   0.51896071          NA           NA      NA         NA
22     22289532  63536292   16   0.36933771          NA           NA      NA         NA
32     22289533  63536292   24   0.18706907          NA           NA      NA 0.18706907
72           NA  63536292   60           NA  0.00000000           NA      NA         NA
42           NA  63536292   64           NA  0.00000000           NA      NA         NA
62           NA  63536292   96           NA          NA 4.103996e-04      NA         NA
82           NA  63536292  124           NA          NA 3.795169e-05      NA         NA
52           NA  63536292  130           NA          NA 2.278583e-05      NA         NA
13     22289597 111299859   16   0.58377725          NA           NA      NA         NA
23     22289598 111299859   24   0.37399383          NA           NA      NA         NA
33     22289599 111299859   48   0.09833684          NA           NA      NA 0.09833684
73           NA 111299859   60           NA  0.04875000           NA      NA         NA
43           NA 111299859   64           NA  0.03250000           NA      NA         NA
63           NA 111299859   96           NA          NA 6.798606e-03      NA         NA
83           NA 111299859  124           NA          NA 1.430805e-03      NA         NA
53           NA 111299859  130           NA          NA 1.024574e-03      NA         NA
14     22289765 112146587   12   1.59483827          NA           NA      NA         NA
24     22289766 112146587   16   1.14287770          NA           NA      NA         NA
34     22289767 112146587   24   0.58690253          NA           NA      NA         NA
44     22289768 112146587   48   0.07948124          NA           NA      NA 0.07948124
84           NA 112146587   60           NA  0.03880000           NA      NA         NA
54           NA 112146587   64           NA  0.02586667           NA      NA         NA
74           NA 112146587   96           NA          NA 1.457679e-03      NA         NA
91           NA 112146587  124           NA          NA 1.414633e-04      NA         NA
64           NA 112146587  130           NA          NA 8.581603e-05      NA         NA
15     22290452 123811839   12  44.59385840          NA           NA      NA         NA
25     22290453 123811839   16  36.10132904          NA           NA      NA         NA
35     22290454 123811839   24  23.66025768          NA           NA      NA         NA
45     22290455 123811839   48   6.66052265          NA           NA      NA         NA
10           NA 123811839   60           NA  2.88000000           NA      NA         NA
85           NA 123811839   64           NA  2.28000000           NA      NA         NA
55     22290456 123811839   72   1.87498220          NA           NA      NA         NA
65     22290457 123811839   96   0.52782018          NA           NA      NA         NA
75     22290458 123811839  120   0.14858495          NA           NA      NA 0.14858495
111          NA 123811839  124           NA          NA 1.202882e-01      NA         NA
92           NA 123811839  130           NA          NA 8.761837e-02      NA         NA
16     22291157 145593586   48  18.32531818          NA           NA      NA         NA
76           NA 145593586   60           NA 12.45500000           NA      NA         NA
56           NA 145593586   64           NA 10.27333333           NA      NA         NA
26     22291158 145593586   72   5.87505987          NA           NA      NA         NA
36     22291159 145593586   96   1.88353229          NA           NA      NA         NA
46     22291160 145593586  120   0.60385664          NA           NA      NA 0.60385664
86           NA 145593586  124           NA          NA 4.995673e-01      NA         NA
66           NA 145593586  130           NA          NA 3.759102e-01      NA         NA
17     22290934 202367199   16 155.89036680          NA           NA      NA         NA
27     22290935 202367199   24 107.16512307          NA           NA      NA         NA
37     22290936 202367199   48  34.81416886          NA           NA      NA         NA
93           NA 202367199   60           NA 18.30500000           NA      NA         NA
77           NA 202367199   64           NA 15.20666667           NA      NA         NA
47     22290937 202367199   72  11.30989560          NA           NA      NA         NA
57     22290938 202367199   96   3.67418619          NA           NA      NA         NA
67     22290939 202367199  120   1.19361351          NA           NA      NA 1.19361351
101          NA 202367199  124           NA          NA 9.896482e-01      NA         NA
87           NA 202367199  130           NA          NA 7.471477e-01      NA         NA
18     22290518 233891605   16  41.50390320          NA           NA      NA         NA
28     22290519 233891605   24  27.33064016          NA           NA      NA         NA
38     22290520 233891605   48   7.80427191          NA           NA      NA         NA
94           NA 233891605   60           NA  4.08000000           NA      NA         NA
78           NA 233891605   64           NA  3.32000000           NA      NA         NA
48     22290521 233891605   72   2.22851202          NA           NA      NA         NA
58     22290522 233891605   96   0.63635223          NA           NA      NA         NA
68     22290523 233891605  120   0.18171056          NA           NA      NA 0.18171056
102          NA 233891605  124           NA          NA 1.474554e-01      NA         NA
88           NA 233891605  130           NA          NA 1.077907e-01      NA         NA
19     22289791 257914734   12   1.66768192          NA           NA      NA         NA
29     22289792 257914734   16   1.47132691          NA           NA      NA         NA
39     22289793 257914734   24   1.14525209          NA           NA      NA         NA
49     22289794 257914734   48   0.54010330          NA           NA      NA         NA
95           NA 257914734   60           NA  0.44800000           NA      NA         NA
69           NA 257914734   64           NA  0.37333333           NA      NA         NA
59     22289795 257914734   72   0.25471386          NA           NA      NA 0.25471386
89           NA 257914734   96           NA          NA 1.201236e-01      NA         NA
103          NA 257914734  124           NA          NA 4.998043e-02      NA         NA
79           NA 257914734  130           NA          NA 4.141846e-02      NA         NA
110    22290260 355441192   48   2.64457864          NA           NA      NA         NA
710          NA 355441192   60           NA  1.82600000           NA      NA         NA
510          NA 355441192   64           NA  1.53800000           NA      NA         NA
210    22290261 355441192   72   0.96307502          NA           NA      NA         NA
310    22290262 355441192   96   0.35072260          NA           NA      NA         NA
410    22290263 355441192  120   0.12772249          NA           NA      NA 0.12772249
810          NA 355441192  124           NA          NA 1.079323e-01      NA         NA
610          NA 355441192  130           NA          NA 8.384501e-02      NA         NA
112    22290686 409149207   12  92.37008404          NA           NA      NA         NA
211    22290687 409149207   16  75.56168235          NA           NA      NA         NA
311    22290688 409149207   24  50.56408802          NA           NA      NA         NA
411    22290689 409149207   48  15.15178707          NA           NA      NA         NA
104          NA 409149207   60           NA  5.93000000           NA      NA         NA
811          NA 409149207   64           NA  4.84666667           NA      NA         NA
511    22290690 409149207   72   4.54031034          NA           NA      NA         NA
611    22290691 409149207   96   1.36052717          NA           NA      NA         NA
711    22290692 409149207  120   0.40768891          NA           NA      NA 0.40768891
113          NA 409149207  124           NA          NA 3.335026e-01      NA         NA
96           NA 409149207  130           NA          NA 2.467485e-01      NA         NA
114    22290960 427589065   16 140.81814484          NA           NA      NA         NA
212    22290961 427589065   24  96.88854127          NA           NA      NA         NA
312    22290962 427589065   48  31.55831549          NA           NA      NA         NA
97           NA 427589065   60           NA 15.79000000           NA      NA         NA
712          NA 427589065   64           NA 13.12000000           NA      NA         NA
412    22290963 427589065   72  10.27910281          NA           NA      NA         NA
512    22290964 427589065   96   3.34808601          NA           NA      NA         NA
612    22290965 427589065  120   1.09053097          NA           NA      NA 1.09053097
105          NA 427589065  124           NA          NA 9.045757e-01      NA         NA
812          NA 427589065  130           NA          NA 6.833690e-01      NA         NA
115    22289816 443820693    8   2.66007299          NA           NA      NA         NA
213    22289817 443820693   12   2.22883711          NA           NA      NA         NA
313    22289818 443820693   16   1.86751073          NA           NA      NA         NA
413    22289819 443820693   24   1.31109047          NA           NA      NA         NA
513    22289820 443820693   48   0.45367131          NA           NA      NA         NA
 [ reached 'max' / getOption("max.print") -- omitted 292 rows ]
> e_off
    PKDATAROWID     SDEID TIME     CEST_KEL CEST_INT     CEST_EXT CEST_C0 CEST_TLAST
1      22289754   2925168   24   1.70263761       NA           NA      NA         NA
2      22289755   2925168   48   0.58780724       NA           NA      NA         NA
3      22289756   2925168   72   0.20293065       NA           NA      NA 0.20293065
5            NA   2925168   96           NA       NA 7.005842e-02      NA         NA
6            NA   2925168  124           NA       NA 2.025778e-02      NA         NA
4            NA   2925168  130           NA       NA 1.552816e-02      NA         NA
11     22289571  40876220   16   0.49544432       NA           NA      NA         NA
21     22289572  40876220   24   0.36517471       NA           NA      NA         NA
31     22289573  40876220   48   0.14622386       NA           NA      NA         NA
41     22289574  40876220   72   0.05855120       NA           NA      NA 0.05855120
61           NA  40876220   96           NA       NA 2.344517e-02      NA         NA
7            NA  40876220  124           NA       NA 8.059791e-03      NA         NA
51           NA  40876220  130           NA       NA 6.411404e-03      NA         NA
12     22289531  63536292   12   0.51896071       NA           NA      NA         NA
22     22289532  63536292   16   0.36933771       NA           NA      NA         NA
32     22289533  63536292   24   0.18706907       NA           NA      NA 0.18706907
52           NA  63536292   96           NA       NA 4.103996e-04      NA         NA
62           NA  63536292  124           NA       NA 3.795169e-05      NA         NA
42           NA  63536292  130           NA       NA 2.278583e-05      NA         NA
13     22289597 111299859   16   0.58377725       NA           NA      NA         NA
23     22289598 111299859   24   0.37399383       NA           NA      NA         NA
33     22289599 111299859   48   0.09833684       NA           NA      NA 0.09833684
53           NA 111299859   96           NA       NA 6.798606e-03      NA         NA
63           NA 111299859  124           NA       NA 1.430805e-03      NA         NA
43           NA 111299859  130           NA       NA 1.024574e-03      NA         NA
14     22289765 112146587   12   1.59483827       NA           NA      NA         NA
24     22289766 112146587   16   1.14287770       NA           NA      NA         NA
34     22289767 112146587   24   0.58690253       NA           NA      NA         NA
44     22289768 112146587   48   0.07948124       NA           NA      NA 0.07948124
64           NA 112146587   96           NA       NA 1.457679e-03      NA         NA
71           NA 112146587  124           NA       NA 1.414633e-04      NA         NA
54           NA 112146587  130           NA       NA 8.581603e-05      NA         NA
15     22290452 123811839   12  44.59385840       NA           NA      NA         NA
25     22290453 123811839   16  36.10132904       NA           NA      NA         NA
35     22290454 123811839   24  23.66025768       NA           NA      NA         NA
45     22290455 123811839   48   6.66052265       NA           NA      NA         NA
55     22290456 123811839   72   1.87498220       NA           NA      NA         NA
65     22290457 123811839   96   0.52782018       NA           NA      NA         NA
72     22290458 123811839  120   0.14858495       NA           NA      NA 0.14858495
9            NA 123811839  124           NA       NA 1.202882e-01      NA         NA
8            NA 123811839  130           NA       NA 8.761837e-02      NA         NA
16     22291157 145593586   48  18.32531818       NA           NA      NA         NA
26     22291158 145593586   72   5.87505987       NA           NA      NA         NA
36     22291159 145593586   96   1.88353229       NA           NA      NA         NA
46     22291160 145593586  120   0.60385664       NA           NA      NA 0.60385664
66           NA 145593586  124           NA       NA 4.995673e-01      NA         NA
56           NA 145593586  130           NA       NA 3.759102e-01      NA         NA
17     22290934 202367199   16 155.89036680       NA           NA      NA         NA
27     22290935 202367199   24 107.16512307       NA           NA      NA         NA
37     22290936 202367199   48  34.81416886       NA           NA      NA         NA
47     22290937 202367199   72  11.30989560       NA           NA      NA         NA
57     22290938 202367199   96   3.67418619       NA           NA      NA         NA
67     22290939 202367199  120   1.19361351       NA           NA      NA 1.19361351
81           NA 202367199  124           NA       NA 9.896482e-01      NA         NA
73           NA 202367199  130           NA       NA 7.471477e-01      NA         NA
18     22290518 233891605   16  41.50390320       NA           NA      NA         NA
28     22290519 233891605   24  27.33064016       NA           NA      NA         NA
38     22290520 233891605   48   7.80427191       NA           NA      NA         NA
48     22290521 233891605   72   2.22851202       NA           NA      NA         NA
58     22290522 233891605   96   0.63635223       NA           NA      NA         NA
68     22290523 233891605  120   0.18171056       NA           NA      NA 0.18171056
82           NA 233891605  124           NA       NA 1.474554e-01      NA         NA
74           NA 233891605  130           NA       NA 1.077907e-01      NA         NA
19     22289791 257914734   12   1.66768192       NA           NA      NA         NA
29     22289792 257914734   16   1.47132691       NA           NA      NA         NA
39     22289793 257914734   24   1.14525209       NA           NA      NA         NA
49     22289794 257914734   48   0.54010330       NA           NA      NA         NA
59     22289795 257914734   72   0.25471386       NA           NA      NA 0.25471386
75           NA 257914734   96           NA       NA 1.201236e-01      NA         NA
83           NA 257914734  124           NA       NA 4.998043e-02      NA         NA
69           NA 257914734  130           NA       NA 4.141846e-02      NA         NA
110    22290260 355441192   48   2.64457864       NA           NA      NA         NA
210    22290261 355441192   72   0.96307502       NA           NA      NA         NA
310    22290262 355441192   96   0.35072260       NA           NA      NA         NA
410    22290263 355441192  120   0.12772249       NA           NA      NA 0.12772249
610          NA 355441192  124           NA       NA 1.079323e-01      NA         NA
510          NA 355441192  130           NA       NA 8.384501e-02      NA         NA
111    22290686 409149207   12  92.37008404       NA           NA      NA         NA
211    22290687 409149207   16  75.56168235       NA           NA      NA         NA
311    22290688 409149207   24  50.56408802       NA           NA      NA         NA
411    22290689 409149207   48  15.15178707       NA           NA      NA         NA
511    22290690 409149207   72   4.54031034       NA           NA      NA         NA
611    22290691 409149207   96   1.36052717       NA           NA      NA         NA
76     22290692 409149207  120   0.40768891       NA           NA      NA 0.40768891
91           NA 409149207  124           NA       NA 3.335026e-01      NA         NA
84           NA 409149207  130           NA       NA 2.467485e-01      NA         NA
112    22290960 427589065   16 140.81814484       NA           NA      NA         NA
212    22290961 427589065   24  96.88854127       NA           NA      NA         NA
312    22290962 427589065   48  31.55831549       NA           NA      NA         NA
412    22290963 427589065   72  10.27910281       NA           NA      NA         NA
512    22290964 427589065   96   3.34808601       NA           NA      NA         NA
612    22290965 427589065  120   1.09053097       NA           NA      NA 1.09053097
85           NA 427589065  124           NA       NA 9.045757e-01      NA         NA
77           NA 427589065  130           NA       NA 6.833690e-01      NA         NA
113    22289816 443820693    8   2.66007299       NA           NA      NA         NA
213    22289817 443820693   12   2.22883711       NA           NA      NA         NA
313    22289818 443820693   16   1.86751073       NA           NA      NA         NA
413    22289819 443820693   24   1.31109047       NA           NA      NA         NA
513    22289820 443820693   48   0.45367131       NA           NA      NA         NA
613    22289821 443820693   72   0.15698204       NA           NA      NA 0.15698204
86           NA 443820693   96           NA       NA 5.431986e-02      NA         NA
92           NA 443820693  124           NA       NA 1.574897e-02      NA         NA
78           NA 443820693  130           NA       NA 1.207895e-02      NA         NA
114    22290077 463304641   12   5.87824590       NA           NA      NA         NA
214    22290078 463304641   16   5.06153051       NA           NA      NA         NA
314    22290079 463304641   24   3.75275385       NA           NA      NA         NA
414    22290080 463304641   48   1.52951851       NA           NA      NA         NA
514    22290081 463304641   72   0.62338938       NA           NA      NA 0.62338938
79           NA 463304641   96           NA       NA 2.540762e-01      NA         NA
87           NA 463304641  124           NA       NA 8.916673e-02      NA         NA
614          NA 463304641  130           NA       NA 7.124493e-02      NA         NA
115    22289975 471081571   24   2.72023498       NA           NA      NA         NA
215    22289976 471081571   48   0.89584521       NA           NA      NA         NA
315    22289977 471081571   72   0.29502548       NA           NA      NA 0.29502548
515          NA 471081571   96           NA       NA 9.715968e-02      NA         NA
615          NA 471081571  124           NA       NA 2.658992e-02      NA         NA
415          NA 471081571  130           NA       NA 2.014298e-02      NA         NA
116    22290778 499820574   16 119.86811280       NA           NA      NA         NA
216    22290779 499820574   24  85.89601519       NA           NA      NA         NA
316    22290780 499820574   48  31.60683899       NA           NA      NA         NA
416    22290781 499820574   72  11.63025163       NA           NA      NA         NA
516    22290782 499820574   96   4.27954067       NA           NA      NA         NA
616    22290783 499820574  120   1.57472675       NA           NA      NA 1.57472675
88           NA 499820574  124           NA       NA 1.333030e+00      NA         NA
710          NA 499820574  130           NA       NA 1.038226e+00      NA         NA
 [ reached 'max' / getOption("max.print") -- omitted 196 rows ]
> 
> summary(r_off$AUC0_64)
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
   15.18   167.13  1226.99  3502.11  4813.88 14776.16 
> summary(r_off$AUC60_124)
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  0.001   5.345  34.425  82.790 113.870 586.860 

```

**Comment by tensfeldt** [comment]
#46 remains an issue with commit 96f6417

**Comment by jhhughes256** [comment]
### Update

I was able to replicate this issue on 18821dca00 making the following changes to the base tc010. Before running the INCLUDEINTERPOLATION test.

```
mct$AUC.4.T1 <- 12
mct$AUC.4.T2 <- 130
mct$AUC.5.T1 <- 24
mct$AUC.5.T2 <- 96
mct$AUC.6.T1 <- 60
mct$AUC.6.T2 <- 124
```

**Comment by rudraya** [comment]
This issue was addressed in 7fd866a, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
This issue is resolve with commit 22d43ea and can be closed.
